src/utils/tests/date.spec.ts

```diff
@@ -1,18 +1,24 @@
 import { monthsDelta } from '../date';
 
+type TestCase = [Date, Date, number];
+
 describe('monthsDelta', () => {
-    it.each`
-        firstDate                                   | secondDate                                    | expected
-        ${new Date(2020, 0, 1)}  | ${new Date(2020, 0, 1)}    | ${0}
-        ${new Date(2020, 3, 1)}  | ${new Date(2020, 0, 1)}    | ${-3}
-        ${new Date(2020, 0, 1)}  | ${new Date(2020, 3, 1)}    | ${3}
-        ${new Date(2020, 0, 15)}  | ${new Date(2020, 3, 17)}  | ${3}
-        ${new Date(2020, 0, 15)}  | ${new Date(2020, 4, 1)}   | ${3}
-        ${new Date(2020, 0, 15)}  | ${new Date(2020, 3, 13)}  | ${2}
-        ${new Date(2020, 0, 30)}  | ${new Date(2020, 1, 1)}  | ${0}
-    `('should round $value with a precision of $precision digits', ({ firstDate, secondDate, expected }) => {
-        const delta = monthsDelta(firstDate, secondDate);
+    const testCases: TestCase[] = [
```

Comment:
Est-ce qu'il y a une raison pour avoir changé de `it.each` à de quoi de custom?

Reply:
Aucune raison, tu préfères `it.each` ?

Reply:
Hmm, je pense que oui, ça me semble être moins de bruit. Mais bon, les deux fonctionnent.

Reply:
Fixed